host/class/uac/usb_host_uac/uac_host.c

```diff
@@ -157,6 +157,9 @@ typedef struct uac_interface {
     uint32_t ringbuf_size;                     /*!< Ring buffer size */
     uint32_t ringbuf_threshold;                /*!< Ring buffer threshold */
     uac_host_dev_info_t dev_info;              /*!< USB device parameters */
+    uint16_t vol_min_db;                       /*!< volume min with 1/256 db step */
+    uint16_t vol_max_db;                       /*!< volume max with 1/256 db step */
+    uint16_t vol_res_db;                       /*!< volume resolution with 1/256 db step */
```

Comment:
Is there any reason for using unsigned type for signed number?

Reply:
@tore-espressif To be honest, this is because I mistakenly used the `uint32_t` type in `uac_host_device_set_volume_db` in the past, so I continued using unsigned type to maintain consistency in calculations.

Reply:
Or, how about making a "break change" to the `uac_host_device_set_volume_db` function, changing the param from uint32_t to int16_t? This would solve all the problems and make the calculations much simpler.

Reply:
I'd prefer introducing a breaking change and having a cleaner code :)

Reply:
@tore-espressif fixed in c2a3782b0f011edf2fb78814175ede1b035d27f8